onnx/defs/parser.cc

```diff
@@ -457,8 +455,6 @@ Status OnnxParser::ParseSingleAttributeValue(AttributeProto& attr, AttributeProt
         attr.set_type(AttributeProto_AttributeType_STRING);
         attr.set_s(literal.value);
         break;
-      default:
```

Comment:
To me this default handle helps if a new enum value is added. Does clang flag this as an error?